opsdroid/cli/version.py

```diff
@@ -8,7 +8,19 @@
 
 
 def print_version(ctx, param, value):
-    """[Deprecated] Print out the version of opsdroid that is installed."""
+    """[Deprecated] Print out the version of opsdroid that is installed.
+
+    Args:
+        ctx (:obj:`click.Context`): The current click cli context
```

Comment:
```suggestion
        ctx (:obj:`click.Context`): The current click cli context.
```

Reply:
Should be resolved in the latest change.